vendor/vendor.json

```diff
@@ -2,12 +2,31 @@
 	"comment": "",
 	"ignore": "test",
 	"package": [
+		{
+			"checksumSHA1": "XiHV3bYcd1RJ03nIT/U4QFynF6w=",
+			"path": "github.com/apex/log",
+			"revision": "3edb93e9f62c13dccdbf0759bd9da5b71dc60926",
+			"revisionTime": "2020-08-18T07:36:17Z",
+			"version": "v1.9.0",
+			"versionExact": "v1.9.0"
+		},
 		{
 			"checksumSHA1": "hwGdeQbcfc2RvIQS5wAaYRKJDd4=",
 			"path": "github.com/imdario/mergo",
 			"revision": "50d4dbd4eb0e84778abe37cefef140271d96fade",
 			"revisionTime": "2016-05-17T06:44:35Z"
+		},
+		{
+			"checksumSHA1": "Qo2E/26skb9mZQ3b2Mh6QDkpBLs=",
+			"path": "github.com/pkg/errors",
+			"revision": "5dd12d0cfe7f152f80558d591504ce685299311e",
+			"revisionTime": "2020-12-14T06:45:52Z"
+		},
+		{
+			"checksumSHA1": "zamq7bor1Nt+tXE3sJ3lH21K/f4=",
+			"path": "log",
+			"revision": ""
 		}
 	],
-	"rootPath": "github.com/pavel-z1/phpipam-sdk-go"
+	"rootPath": "phpipam-sdk-go"
```

Comment:
Return please root path to "github.com/pavel-z1/phpipam-sdk-go"

Reply:
Oups, sorry, fixing